### PR TITLE
secure party change modal and ux items

### PR DESCRIPTION
**Pull request by jordiwes**
*Issue #:* /bcgov/entity#7864

*Description of changes:*
- some ux updates for other registrations
- secured party change modal

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-ui/src/components/parties/SecuredParties.vue [hunk 1]
```diff
@@ -365,6 +381,7 @@ export default defineComponent({
     const fetchOtherSecuredParties = async () => {
       localState.loading = true
       if (parties.registeringParty) {
+        parties.registeringParty.businessName = 'Bank of Montreal'
```
Comment (doug-lovett) [inline ppr-ui/src/components/parties/SecuredParties.vue]: Should this be removed?

Comment (jordiwes) [reply ppr-ui/src/components/parties/SecuredParties.vue]: whoops!

**Review by doug-lovett — APPROVED** [review]

**Review by doug-lovett — APPROVED** [review]
Adding OT other type description looks good.

**Review by kialj876 — CHANGES_REQUESTED** [review]
Looks good to me, but can you add in some tests please? New dialogue should have a test suite and we should probably test it coming up wherever its needed (should be pretty similar to existing tests we have for the confirmation dialog).
I also think you could've used the confirmation dialog for this one (would've needed an 'if' around the checkbox is the only change right?), but it doesn't really matter. Would've been less test setup for ya though!

**Comment by jordiwes** [comment]
I added some tests :)

**Review by kialj876 — APPROVED** [review]
Thanks for adding those in!

**Pull request merged**